zerver/webhooks/stripe/view.py

```diff
@@ -198,6 +186,40 @@ def default_body(update_blacklist: List[str]=[]) -> str:
             amount=amount,
             end=end
         )
+
+    if category.startswith('invoice'):
+        object_id = object_['id']
+        link = "https://dashboard.stripe.com/invoices/{}".format(object_id)
+        amount = amount_string(object_["amount_due"], object_["currency"])
+        body_t = "An {invoice_type} **[invoice]({link})** for the payment of amount **{amount}** {end}."
+
+        INVOICE_INFO = {"created": "has been created",
+                        "deleted": "has been deleted",
+                        "finalized": "has been finalized",
+                        "marked_uncollectible": "has been marked as uncollectible",
+                        "payment_failed": "has failed",
+                        "payment_succeeded": "has succeeded",
+                        "sent": "has been sent",
+                        "upcoming": "has been created",
+                        "updated": "has been updated",
+                        "voided": "has been voided"
+                        }
```

Comment:
I would move this dictionary to the top of the file after the exceptions classes.
Also, format the dictionary like so:
```
INVOICE_INFO = {
    "created": "has been created",
    ... and so on
}
```

Reply:
I've moved the dictionary to the top of the `if category.startswith('invoice'):` because if we move it at the top as you said then it will create problems. One of the problems can be that there are other events such as payout created, and there might be confusion with payout and invoice created because we have first checked for the invoice category and then went ahead with the event type. I hope you understood it.

Reply:
@sameerchoubey: Well, not really, you can still move it to the top where I suggested. The main reason for doing so is that the `INVOICE_INFO` is a constant, and it is a popular Python convention to move constants to the top of the file right after the imports.

Also, the `INVOICE_INFO` dictionary should only be used for invoice events, so it wouldn't be used for payout events at all, so I don't see what you mean by it affecting payout events. :) I'll leave a comment in the file explaining it more clearly.